### DISCUSSION TITLE
Laravel Fortify Integration

### DISCUSSION THREAD

**Discussion by reiquanMBA** [orchidsoftware/platform#1590] [Q&A] [answered]
It came to my attention that 2fa was removed from Laravel Orchid. Is there any documentation i can find that can help me implement Laravel Fortify inside of the Orchid Platform? Thanks in advance

**Comment by tabuna** [comment, marked as answer]
Hi, for this, we have prepared: https://github.com/orchidsoftware/fortify

**Reply by reiquanMBA** [reply]
thank you so much!
